CMakeLists.txt

```diff
@@ -147,10 +147,20 @@ endif()
 message(STATUS "Current install path is: ${CMAKE_INSTALL_PREFIX}")
 
 # Generate version numbers
+find_package(Git QUIET)
+execute_process(COMMAND
+  "${GIT_EXECUTABLE}" describe --tags --always --dirty
+  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
```

Comment:
This should use CMAKE_CURRENT_SOURCE_DIR to support building p-net as a subproject.

Reply:
Done